code-challenges.test.js

```diff
@@ -15,37 +15,189 @@
+5.) return tallied votes
+
+*/
+
+// const votes1 = { upVotes: 13, downVotes: 2 }
+// // Expected output: 11
+// const votes2 = { upVotes: 2, downVotes: 33 }
+// // Expected output: -31
+
+const talliedVotes = (votes) => {
+  return votes.upVotes - votes.downVotes
+}
+
+/*
+
+ PASS  ./code-challenges.test.js
+  talliedVotes
+    ✓ return the number of votes tallied (1 ms)
+
+*/
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const dataArray1 = ["array", "object", "number", "string", "Boolean"]
-const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
-// Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
+describe("noDups", () => {
+  const dataArray1 = ["array", "object", "number", "string", "Boolean"]
+  const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
+  // Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
+  it("return an array with no duplicate values", () => {
+    expect(noDups(dataArray1, dataArray2)).toEqual(["array", "object", "number", "string", "Boolean", "null", "undefined"])
+  })
+})
+
+/*
+
+ FAIL  ./code-challenges.test.js
+  rfiShuffles
+    ✓ return an array whos content is a subset of the incoming array with the first item removed and the remaining content shuffled (1 ms)
+  talliedVotes
+    ✓ return the number of votes tallied
+  noDups
+    ✕ return an array with no duplicate values (1 ms)
+
+  ● noDups › return an array with no duplicate values
+
+    ReferenceError: noDups is not defined
+
+  */
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // b) Create the function that makes the test pass.
+
+/*
+
+Pseudocode: 1.) Create an function called noDups
+2.) noDups will take in two arrays as arguments
+3.) use the ... spread operator to make deep copies of the dataArray1 and dataArray2, then store them in combinedArray variable
+4.) use ...new Set to create a deep copy of combinedArray and store it in filteredArray.  ...new Set only allows a value to occur once in the copy and will fill filter out duplicate values in combinedArray.
+5.) return filteredArray
+
+*/
+
+// const dataArray1 = ["array", "object", "number", "string", "Boolean"]
+// const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
+// // Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
+
+const noDups = (array1, array2) => {
+  
+  let combinedArray = [...array1, ...array2];
+  let  filteredArray = [...new Set(combinedArray)];
+  return filteredArray
+}
+
```

Comment:
Great job using `new Set()` to filter out dupes! One note: there's no need to create the `filteredArray` variable at all, you can just return the logic!